lib/screens/sub_category_list_view.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:flutter/material.dart';
+
+import '../db/entity/category_topic.dart';
+import '../repos/category_topic_repo.dart';
+
+class SubcategoryList extends StatefulWidget {
+  final String categoryName;
+  final String categoryId;
+
+  const SubcategoryList({Key key, this.categoryName, this.categoryId})
+      : super(key: key);
+  @override
+  _SubcategoryListState createState() => new _SubcategoryListState();
+}
+
+class _SubcategoryListState extends State<SubcategoryList> {
+  @override
+  List<CategoryTopic> _dataCategory = new List<CategoryTopic>();
```

Comment:
you have 2 class variables: _dataCategory and _categoryData. Both of which are assigned to the same. Do you need both of these?